### PR TITLE
api: Support clean up ttl setting

### LINKED ISSUE #4801: Can't clean up the TTL setting
### What did you do?
As described #4780, we need to clean up the TTL setting firstly when we want to change the config which has already had a TTL setting. So I try to set TTL as 0.

### What did you expect to see?
clean up the TTL setting

### What did you see instead?
It didn't work.

kEcv4hKbKP

### What version of PD are you using (`pd-server -V`)?
master

### PR DESCRIPTION
**Pull request by CabinfeverB**
Signed-off-by: Cabinfever_B <cabinfeveroier@gmail.com>

### What problem does this PR solve?

Issue Number: Close #4801

### What is changed and how does it work?
Check if the TTL parameter is extracted

```commit-message
Support clean up ttl setting
```

### Check List

Tests

- Unit test

### Release note

```release-note
None.
```

### CODE REVIEW

File: server/api/config.go [hunk 1]
```diff
@@ -99,17 +99,19 @@ func (h *confHandler) SetConfig(w http.ResponseWriter, r *http.Request) {
 	}
 
 	var ttls int
+	var ttlFlag bool
```
Comment (rleungx) [inline server/api/config.go:102]: How about using `ttlSec` directly?

**Review by lhy1024 — APPROVED** [review]

File: server/api/config.go [hunk 2]
```diff
@@ -98,19 +98,15 @@ func (h *confHandler) SetConfig(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	var ttls int
 	if ttlSec := r.URL.Query().Get("ttlSecond"); ttlSec != "" {
 		var err error
-		ttls, err = strconv.Atoi(ttlSec)
+		ttls, err := strconv.Atoi(ttlSec)
```
[the remark this answers is not in the document]
Comment (CabinfeverB) [reply server/api/config.go:102]: done

**Review by rleungx — APPROVED** [review]

**Pull request merged**